test/real_eq_operator_test.cpp

```diff
@@ -162,17 +162,17 @@ TEST_CASE("Operator ==") {
 
         SECTION("With precision exception") {
```

Comment:
Could I not just add a check for the EXPLICIT case? And for algorithm numbers, what precision to use for comparison as it can be arbitrarily high if not specified by set_precision